grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -1039,7 +1039,7 @@ class OrderService {
     }
 
     def canOrderItemBeEdited(OrderItem orderItem, User user) {
-        def isPending = orderItem?.order?.status == OrderStatus.PENDING
+        def isPending = orderItem?.order?.status < OrderStatus.PENDING
```

Comment:
actually why didn't it work before? There seems not to be any "lower" status than pending? :thinking:

Reply:
This one baffles me a little bit. This does not make sense to me. Could you try using `order.isPending` instead?